### DISCUSSION TITLE
Compatibility with the new StartAllBack taskbar implementation

### DISCUSSION THREAD

**Discussion by nicolafarina** [ramensoftware/windhawk-mods#890] [General]
Hi there,

is there a chance to have full compatibility with the new taskbar implementation of StartAllBack? 7+ Taskbar Tweaker and Windhawk don't work either (for taskbar enhancements). I really need middle-click custom action (close window) with StartAllBack, because it allows having the taskbar on vertical right with complete functionalities.

Thanks.

**Comment by m417z** [comment]
Hi,
I didn’t look at the new StartAllBack versions, but as far as I understand, the dormant Win10 taskbar code was removed in Win11 24H2, and so both ExplorerPatcher and StartAllBack came up with a custom taskbar implementation. That means that both 7+ Taskbar Tweaker and Windhawk mods have to be adjusted/reimplemented for the custom version of the taskbar, which is a huge project and not something I’m planning to do at this point.
Note that as long as you didn't update to Win11 24H2, you can keep using an older version of StartAllBack, which I believe should work.

**Comment by nicolafarina** [comment]
Thanks for your effort in replying, though.
